src/Microsoft.Framework.PackageManager/Install/InstallGlobalCommand.cs

```diff
@@ -67,6 +67,17 @@ public FeedOptions FeedOptions
 
             var temporaryProjectFileFullPath = CreateTemporaryProject(FeedOptions.TargetPackagesFolder, packageId, packageVersion);
 
+            var packageFullPath = Path.Combine(
+                _commandsRepository.PackagesRoot.Root,
+                _commandsRepository.PathResolver.GetPackageDirectory(packageId, new SemanticVersion(packageVersion)));
+
+            if (OverwriteCommands)
+            {
+                WriteVerbose($"Deleting {packageFullPath}");
+                FileOperationUtils.DeleteFolder(packageFullPath);
```

Comment:
Um... what happens if the packages is not installed? As in running with `--overwrite` when there is nothing to overwrite